### PR TITLE
Weather card

### PR DESCRIPTION
**Pull request by agnieszkazbikowska-madetech**
Create a component that displays a day weather.

### CODE REVIEW

**Comment by agnieszkazbikowska-madetech** [comment]
[Create a component to display a day's weather with tests](https://trello.com/c/RSfhpgkQ/8-create-a-component-to-display-a-days-weather-with-tests)

File: src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+// import { render, screen } from '@testing-library/react';
```
Comment (agnieszkazbikowska-madetech) [inline src/App.test.js]: We can delete these tests.

File: src/components/dayCard/DayCard.test.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    expect(dayOfTheWeek).toBeInTheDocument();
+  });
+
+  test('it renders the date', () => {
+    render(<DayCard day='Saturday' date='March 2nd' />);
+    const dayOfTheWeek = screen.getByText('March 2nd');
+    expect(dayOfTheWeek).toBeInTheDocument();
+  });
+
+  test('it renders the correct icon', () => {
```
Comment (agnieszkazbikowska-madetech) [inline src/components/dayCard/DayCard.test.js]: Here we have the word `correct`, we don't have on all the tests. Shall we unify it?

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,26 @@
...
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+  display: flex;
```
Comment (agnieszkazbikowska-madetech) [inline src/App.css]: Do we want to keep the styles in the `App` or in a separate `DayCard` file?

Comment (Joseaie) [reply src/App.css]: In a separate `DayCard.css` file would make things clearer, I think.

File: src/App.js [hunk 4]
```diff
@@ -1,8 +1,18 @@
...
-    <div className="App" />
+    <div className='App'>
+      <DayCard
+        day='Saturday'
+        date='March 2nd'
+        temperature='36'
+        description='light rain'
+        imgSrc={Wind}
+      />
+    </div>
```
Comment (Joseaie) [inline src/App.js]: I think the changes in this file should either not have been committed (as it is for testing purposes) or be in a separate PR (as it's implementing the `DayCard` and not part of the creation, which this PR is for)

File: src/App.test.js (hunk 1 above)
Comment (Joseaie) [reply src/App.test.js]: True, but deleting them would also be separate from this PR, I think 😆

File: src/components/dayCard/DayCard.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    ? imgAltText
+    : imgSrc?.substring(0, imgSrc.indexOf('.'));
+
+  return (
+    <div className='day-card'>
+      <h2>{day}</h2>
+      <p className='date'>{date}</p>
+      <img className='weather-icon' src={imgSrc} alt={altText} />
+      <p className='temperature'>
+        <b>{temperature} &deg;C</b>
```
Comment (Joseaie) [inline src/components/dayCard/DayCard.js]: Can this (making the text bold) be a CSS property rather than a HTML tag?

File: src/components/dayCard/DayCard.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+function DayCard({ date, day, imgAltText, imgSrc, temperature, description }) {
```
Comment (Joseaie) [inline src/components/dayCard/DayCard.js]: nit: Very minor, but should this be alphabetised?

File: src/App.test.js (hunk 1 above)
Comment (agnieszkazbikowska-madetech) [reply src/App.test.js]: Yes :D

File: src/App.css (hunk 3 above)
Comment (agnieszkazbikowska-madetech) [reply src/App.css]: That's done!

File: src/components/dayCard/DayCard.js (hunk 5 above)
Comment (agnieszkazbikowska-madetech) [reply src/components/dayCard/DayCard.js]: Done!

File: src/components/dayCard/DayCard.js (hunk 6 above)
Comment (agnieszkazbikowska-madetech) [reply src/components/dayCard/DayCard.js]: It would be nice :-)

**Review by Joseaie — APPROVED** [review]
Looking clean sis! Well done ✨
